sys-apps/plocate/files/plocate-1.1.5-no-systemd-check.patch

```diff
@@ -0,0 +1,29 @@
+diff -abur a/meson.build plocate-1.1.5/meson.build
+--- a/meson.build	2021-03-05 12:40:39.594922524 -0500
++++ b/meson.build	2021-03-05 12:40:44.654922623 -0500
+@@ -76,17 +76,17 @@
+                configuration: conf_data)
+ install_man(updatedb_conf_man)
+ 
+-if get_option('install_systemd')
```

Comment:
Let's drop commented line from patches - we already know what's missing (from the - lines).